CHANGELOG.md

```diff
@@ -1468,7 +1468,7 @@ Changes:
 - Changed: `time-no-imperceptible` now checks vendor prefixed properties.
 - Changed: `unit-*` rules now check `@media` values too.
 - Added: plugins can allow primary option arrays by setting `ruleFunction.primaryOptionArray = true`.
-- Added: [processors](/docs/user-guide/configure.md#processors).
+- Added: processors.
```

Comment:
```suggestion
- Added: [processors](https://github.com/stylelint/stylelint/blob/14.16.1/docs/user-guide/configure.md#processors).
```

Reply:
Thanks for the suggestion. But I feel like it's inconsistent that only this item has a link.